rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -124,6 +141,17 @@ def _decode_curve_deposit_events(
                 event.event_subtype = HistoryEventSubType.RECEIVE_WRAPPED
                 event.counterparty = CPT_CURVE
                 event.notes = f'Receive {event.balance.amount} {event.asset.symbol} after depositing in curve pool'  # noqa: E501
+            elif (  # deposit give asset
+                event.event_type == HistoryEventType.SPEND and
+                event.event_subtype == HistoryEventSubType.NONE and
+                event.location_label == user_address and
+                tx_log.address in self.curve_pools
+            ):
+                event.event_type = HistoryEventType.DEPOSIT
+                event.event_subtype = HistoryEventSubType.DEPOSIT_ASSET
+                event.counterparty = CPT_CURVE
+                event.notes = f'Deposit {event.balance.amount} {event.asset.symbol} in curve pool'  # noqa: E501
```

Comment:
Can you maybe get the curve pool name here somehow? Not a must. Just a question/good to have.

Reply:
I've added it where is possible to know exactly the contract where they were deposited/withdrawn